runtime/compiler/trj9/env/VMJ9.cpp

```diff
@@ -1280,6 +1280,12 @@ TR_J9VMBase::getReferenceFieldAtAddress(uintptrj_t fieldAddress)
    {
    TR_ASSERT(haveAccess(), "Must haveAccess in getReferenceFieldAtAddress");
    TR::Compilation* comp = TR::comp();
+   // Emit read barrier
```

Comment:
Can we push the comment into the `#if` so it doesn't dangle if the file is preprocessed without defining `OMR_GC_CONCURRENT_SCAVENGER`?

Reply:
Done